hbw/app/models/hbw/fields/static.rb

```diff
@@ -3,6 +3,20 @@ module Fields
     class Static < Ordinary
       self.default_data_type = :string
 
+      def as_json
+        result = definition.symbolize_keys.dup
+        result[:html] = prepare_string(result[:html])
+        result
+      end
+
+      def prepare_string(str)
+        str.gsub(/%{.*?}/){|s| prepare_value(s)}
+      end
+
+      def prepare_value(str)
+        task.definition['variables'].select{|v|  v['name'] == str.gsub(/[%\{\}]/, '')}.first['value']
```

Comment:
Two spaces:
```
|v|  v
```

Fix formatting for blocks. This is how we format them https://github.com/latera/homs/blob/master/hbw/lib/hbw/container.rb#L9